include/platform-aws.h

```diff
@@ -72,7 +72,15 @@ class PlatformAWS : public Platform {
 		return fields ? fields->func_idx : -EIO;
 	}
 
+	// Determine AWS at runtime
```

Comment:
This comment is super confusing, since AWS is also the shortened name of this class.